### ISSUE TITLE
[BUG/Help] bias in self identification in role playing tasks

### ISSUE THREAD

**Issue by fangzhangmnm** [THUDM/ChatGLM-6B#782]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The model keep identifying itself as a 猫耳娘 no matter what character it is supposed to play after I finetuned it for roleplaying tasks using the ptuning script in this repo
There is no 猫耳娘 in the training data. I think it might be a bias in the base model

### Expected Behavior

The model identify itself as the character It learned from the training data.

### Steps To Reproduce

Prepare training data. I use chatgpt to convert a novel to a drama script, like {prompt="A: dialogue  B: " label="dialogue"} ~30K training items.
Use the ptuning script in the repo to train the model.
Let it complete the text "A: 请介绍一下你自己   B:"

### Environment

```markdown
- OS:
- Python:
- Transformers: 
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```
